### DISCUSSION TITLE
Simpler instruction guide?

### DISCUSSION THREAD

**Discussion by namkha209** [Geobert/disle#4] [Q&A]
Hello, I am an interested user and a Game Master for a group of players using the OVA tabletop system, and after the unfortunate demise of Sidekick, your project seems to be the last hope for our group. Yet, I cannot install/'host' it as the instruction are very... unintuitive to a person with next to no coding language like me. Could you please update the installation guide so that I can hopefully try it out for my group?

**Comment by Geobert** [comment]
Hi!

Thank you for the intererest into Dìsle!

Have you use a remote server before? Not "server" like in Discord's server, but like in "real server". A remote computer you lend to host programs like a web server or a Discord bot.
Do you have Linux administration experience? 

If not I'm afraid it's not some instruction that you need but a course on Linux remote administration… which is out of scope :(

The alternative would be to run the bot on your computer with the culprit that if your computer is offline, the bot will be as well.

I've pushed a 1.1.2 version which bring Windows binary (I assume you're on Windows) so you can get the bot without using coding tools.

But you still need to follow the instructions from the Readme (which I've updated a bit).

Hope this helps

**Reply by namkha209** [reply]
Thank you so much for your kindness!

Sadly, I have next to no coding knowledge aside from three months education on C++, so I'm afraid I have no such experience, especially with Linux. In fact, I have never left the Windows platform in my life, as sad as that sounds.

And I am ok with running the bot on my own computer, since I do have a habit of letting run overnight anyways. Besides, I don't expect our group to do too much rolling outside of game sessions, which would roughly be once or twice a week.

Great appreciation, thank you for the help and I hope your project gains more success!

**Reply by Geobert** [reply]
Did you manage to get the bot running?
